site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+# Using Kubernetes Cluster Operator
+
+This section covers how to deploy Custom Resource objects that will
+be managed by the Kubernetes Cluster Operator. If you have not installed
+the Cluster Operator, check the [Installing Cluster Operator in a Kubernetes cluster](/install-cluster-operator.html).
+
+## Confirm Service Availability
+
+Before configuring your app to use Cluster Operator, ensure that RabbitmqCluster Custom Resource is deployed
+to your Kubernetes cluster and is available.
+
+To confirm this availability:
+
+1. Run:
+
+    ```
+    kubectl get customresourcedefinitions.apiextensions.k8s.io
+    ```
+
+1. Verify that `rabbitmqclusters.rabbitmq.com` is on the list, as in the example below.
+
+    <pre class="terminal">
+    $ kubectl get customresourcedefinitions.apiextensions.k8s.io
+    NAME                                   CREATED AT
+    rabbitmqclusters.rabbitmq.com   2019-10-23T10:11:06Z
+    </pre>
+
+If it is not, ask your operator to install it by following the steps in [Installing Cluster Operator in a Kubernetes cluster](/install-cluster-operator.html).
+
+
+## <a id='psp' class='anchor' href='#psp'>(Optional) Apply Pod Security Policies</a>
+
+If you have Pod Security Policies enabled in the Kubernetes cluster, you must create a `[Cluster]Role` and
+`[Cluster]RoleBinding` to enable the Pods to be scheduled. For more information about Pod security policies, see the
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/policy/pod-security-policy/). If you chose to use
+`Role` and `RoleBinding`, it will only be effective in the Namespace where the RBACs are deployed.
+
+If you do not have Pod security policies enabled, skip to <a href="#create">Create a RabbitMQ Instance</a> below.
+
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#example) has an example
+to create RBAC rules and a policy.
+
+## <a id='create' class='anchor' href='#create'>Create a RabbitMQ Instance</a>
+
+To create a RabbitMQ instance, you must create a `RabbitmqCluster` resource definition and apply it.
+Cluster Operator creates the necessary resources, such as Services and StatefulSet, in the same namespace
+in which the `RabbitmqCluster` was defined.
+
+1. Create a YAML file to define a `RabbitmqCluster` resource named `definition.yaml`.
+
+    <p class="note">
+      <strong>Note:</strong> The YAML file can have any name, but the steps that follow assume it is named
+      <code>definition</code>.
+    </p>
+
+1. Copy and paste the below snippet into the file and save it.
+
+    <pre class='hljs'>
```

Comment:
Perhaps this should be `class="lang-yaml"`? We don't highlight `lang-yaml` blocks at the moment and I'm not sure if our highlight.js build includes YAML but it won't be difficult to add, see `page.xsl` under

```
// code highlighting
```